examples/persistent-layout/pages/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Link from 'next/link'
+import AppLayout from '../components/AppLayout'
+
+export default class IndexPage extends React.Component {
+  static pageLayout = AppLayout
```

Comment:
Lets make this `layout` instead of `pageLayout`. Both are clear as to what they do, and `layout` is easier to remember 😄 👍

Reply:
ok! thanks for advice!